conan/tools/cmake/toolchain.py

```diff
@@ -373,8 +373,8 @@ class FindConfigFiles(Block):
     def context(self):
         # To find the generated cmake_find_package finders
         # TODO: Change this for parameterized output location of CMakeDeps
-        cmake_prefix_path = "${CMAKE_CURRENT_LIST_DIR}"
-        cmake_module_path = "${CMAKE_CURRENT_LIST_DIR}"
+        cmake_prefix_path = self._conanfile.generators_folder.replace("\\", "/")
```

Comment:
This one is tricky. Apparently it can stay, as the toolchain file will always be generated in the same folder (generator_folder) as the xxxxconfig.cmake scripts, apparently ``${CMAKE_CURRENT_LIST_DIR}`` will always be correct.